Corona-Warn-App/src/main/java/de/rki/coronawarnapp/submission/SubmissionRepository.kt

```diff
@@ -37,7 +37,8 @@ class SubmissionRepository @Inject constructor(
     private val deadmanNotificationScheduler: DeadmanNotificationScheduler,
     private val analyticsKeySubmissionCollector: AnalyticsKeySubmissionCollector
 ) {
-    private val testResultReceivedDateFlowInternal = MutableStateFlow(Date())
+    private val testResultReceivedDateFlowInternal =
+        MutableStateFlow(Date(LocalData.initialTestResultReceivedTimestamp() ?: System.currentTimeMillis()))
```

Comment:
Why should we ever show the current system time? In this state we always have a time when the user received the test result otherwise you won't be in EOL. Or did I understood something wrong?

Reply:
I don't know the reason why it was implemented this way in the beginning.
Another solution would be to change the flow to be nullable, but it could influence more modules.

Reply:
We agreed to this change in a follow-up PR on 1.15 - which goes through full regression testing.